draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -320,66 +464,90 @@ Internet-Draft      Network Slice Topology Data Model       January 2024
-   named as Network Slice Blue and Network Slice Red, are created by
-   separate customers and delivered to the network slice service
-   provider.  The provider maps the two intents to corresponding network
-   resource partitions (NRPs) internally.  In realizing the network
-   resource partitions, node virtualization is used to separate and
-   allocate resources in physical devices.  Two virtual routers VR1 and
-   VR2 are created over physical router R1, and two virtual routers VR3
-   and VR4 are created over physical router R2, respectively.  Each of
-   the virtual routers,as a partition of the physical router, takes a
+Internet-Draft      Network Slice Topology Data Model      February 2024
+
+
+   In the example shown in Figure 2, two topology intents named as
+   Network Slice Blue and Network Slice Red, are created by separate
+   customers and delivered to the network slice service provider.  The
+   provider maps the two intents to corresponding network resource
+   partitions (NRPs) internally.  In realizing the network resource
+   partitions, node virtualization is used to separate and allocate
+   resources in physical devices.  Two virtual routers VR1 and VR2 are
+   created over physical router R1, and two virtual routers VR3 and VR4
+   are created over physical router R2, respectively.  Each of the
+   virtual routers,as a partition of the physical router, takes a
    portion of the resources such as ports and memory in the physical
    router.
    Depending on the requirements and the implementations, they may share
    certain resources such as processors, ASICs, and switch fabric.
 
+   A network slice customer has the capability to configure customer
+   intent topologies without needing any prior knowledge of the
+   provider's network or resource availability.  However, this approach
+   could potentially create challenges for the provider in understanding
+   and realizing the intended topology.
+
+   Alternatively, the provider can choose to describe the available
+   resources and capabilities in the form of an abstract topology, which
+   is then exposed to the customer before network slice requests.  By
+   doing so, the provider empowers the customer to build their
+   customized intent topologies based on this pre-exposed information.
+   This approach streamlines the process, minimizing unnecessary
+   negotiations between the customer and the provider.  The process and
+   the data models for the provider to expose abstract topologies are
+   outside the scope of this document.
+
+   The provider communicates the operational state of the topology,
+   reflecting the allocated resources that result from negotiations
+   between the customer and the provider.  Subsequently, customers can
+   process the requested topology and seamlessly integrate it into their
+   own network topology.  Importantly, this relationship between the
+   customer and provider can be recursive.  For instance, a customer who
```

Comment:
I'm a bit confused here form the process: which network topology is the "own network topology"? Let me try to rephrase a bit ;
" subsequently , customer can process the network topology get form operational datastore (with resource allocation proposal from provider) and integrate that on the customer intent topology ." 
Is correct my proposal?

Reply:
Yes the topology here refers to the customer intent topology, not the abstract topology which is outside the scope of the document.